tests/apple-pay/ApplePaySessionPaymentStepMockObject.js

```diff
@@ -0,0 +1,66 @@
+class ApplePaySession {
```

Comment:
I just noticed this, why is the mock file in JS not TS? 🤔

Reply:
We learned (copied) this mock from [PineapplePay](https://chrome.google.com/webstore/detail/pineapple-pay/pkohlkkjckhbnapmcmodmnimndfcgpll?hl=en) Chrome extension at beginning. 🫣

Reply:
Ok let's convert this into TS 🙂

Reply:
@davidchin Playwright doesn't seem to allow TS files to be added as part of the `initScript` call. doc: https://playwright.dev/docs/api/class-browsercontext#browser-context-add-init-script-option-script

Reply:
The Playwright execution context does allow TS environment but doesn't allow base classes to extend from, nor the ability to get types from a node_module